dash/dash.py

```diff
@@ -443,6 +443,7 @@ def __init__(  # pylint disable=too-many-statements
 
         _get_paths.CONFIG = self.config
         _pages.CONFIG = self.config
+        _get_paths.SERVER = self.server
```

Comment:
We’ll need to cover the case where you create a server later and add it via `app.init_app(server)` - so I think this line should move down after this one:

https://github.com/plotly/dash/blob/693b62cc68701628b4836fc80f6306af5ae50a9b/dash/dash.py#L481

You asked (on slack) about tests - these can be pretty simple, and should go in `tests/unit` somewhere. Just create an app - once the normal way, once passing in a server later - but in each case finish with:

`assert dash.get_server() == app.server`

Reply:
Great - thanks.
I put this in the `_get_paths` folder with the other functions like `dash.get_asset_url` since it's kinda similar.  Should it be somewhere else, or rename this folder to something more generic in case there are other functions that would would like to add here as well?